Ejercicio_2/Sword.java

```diff
@@ -0,0 +1,27 @@
+package Ejercicio_2;
+
+public class Sword extends Weapon{
+    String handle = ""; //No supe a qué se refería con este atributo, así que lo dejé en blanco
+
+    public Sword (String name, double power) {
+        setName(name);
+        setAmount(1);
+        setType("Sword");
+        setPower(power);
+    }
+
+    public void setHandle(String caliber) {
```

Comment:
Change variable name to handle and implement the function `this.handle = handle;`